akka-javasdk/src/main/scala/akka/javasdk/impl/client/ComponentMethodRefImpl.scala

```diff
@@ -60,4 +60,10 @@ private[impl] final case class ComponentMethodRefImpl[A1, R](
     createDeferred(metadataOpt, Some(arg)).asInstanceOf[DeferredCallImpl[NotUsed, R]].invokeAsync()
   }
 
+  // Note: invoke/ask timeout handled by runtime so no timeout needed here
+  override def invoke(): R =
+    invokeAsync().toCompletableFuture.get()
```

Comment:
unwrap ExecutionException so that we throw the original exception
use `ErrorHandling.unwrapExecutionException`

probably same in other places where we use `.get`